docs/external-secrets.md

```diff
@@ -0,0 +1,53 @@
+---
+title: External secrets
+description: Use an external secrets vault with n8n.
+contentType: howto
+---
+
+# External secrets
+
+!!! info "Feature availability"
+	* External secrets are available on Enterprise Self-hosted and Cloud plans.
+	* n8n supports Infisical and HashiCorp Vault. 
+	* n8n doesn't support [HashiCorp Vault Secrets](https://developer.hashicorp.com/hcp/docs/vault-secrets){:target=_blank .external-link}.
+
+You can use an external secrets store to manage credentials for n8n.
+
+## Connect n8n to your secrets store
+
+1. In n8n, go to **Settings** > **External Secrets**.
+1. Select your store provider.
+1. Enter the credentials for your provider:
+	* Infisical: provide a **Service Token**. Refer to Infisical's [Service token](https://infisical.com/docs/documentation/platform/token){:target=_blank .external-link} documentation for information on getting your token. If you self-host Infisical, enter the **Site URL**.
+
+	!!! note "Infisical environment"
+		Make sure you select the correct Infisical environment when creating your token. n8n will load secrets from this environment, and won't have access to secrets in other Infisical environments. n8n only support service tokens that have access to a single environment.
+
+	* HashiCorp Vault: provide the **Base URL** for your vault instance, and select your **Authentication Method**. Enter your authentication details. If you're using a token, refer to HashiCorp's [Token management](https://developer.hashicorp.com/vault/tutorials/tokens/token-management){:target=_blank .external-link} documentation for information on getting your token.
+
```

Comment:
What about adding two additional points:
- Save the provider configuration.
- Enable the provider with the **Disabled / Enabled** toggle.

it is very important that users remember to enable the provider